variables.tf

```diff
@@ -21,13 +21,13 @@ variable "dd_site" {
 variable "datadog_forwarder_version" {
   type        = string
   description = "The Datadog Forwarder version to use"
-  default     = "3.66.0"
+  default     = "3.121.0"
 }
 
 variable "runtime" {
   type        = string
   description = "The version of the runtime to use"
-  default     = "3.8"
+  default     = "3.11"
```

Comment:
tflint is saying this is an invalid runtime version 🤔

Reply:
we probably need to update the lint version, looking into it

Reply:
Yeah it seems like thats probably it because the docs definitely say 3.11 👍 
https://github.com/DataDog/datadog-serverless-functions/tree/master/aws/logs_monitoring#upgrade-an-older-version-to-3990

Reply:
fix it but also just thought why dont we move this to circle